setup.py

```diff
@@ -1,33 +1,31 @@
-"""
-.. See the NOTICE file distributed with this work for additional information
-   regarding copyright ownership.
+# .. See the NOTICE file distributed with this work for additional information
```

Comment:
What are the two dots for ?

Reply:
I'm going to make the changes here and in the `main` branch to follow the latest advice regarding [Ensembl Sofware License](https://www.ebi.ac.uk/seqdb/confluence/display/ENS/Ensembl+Software+License).

Reply:
@JAlvarezJarreta  Yes please do :-)